server/tserver/src/main/java/org/apache/accumulo/tserver/compactions/ExternalCompactionExecutor.java

```diff
@@ -190,10 +190,10 @@ ExternalCompactionJob reserveExternalCompaction(long priority, String compactorI
   }
 
   public Stream<TCompactionQueueSummary> summarize() {
-    HashSet<Long> uniqPrios = new HashSet<Long>();
+    HashSet<Short> uniqPrios = new HashSet<Short>();
     queuedTask.forEach(task -> uniqPrios.add(task.getJob().getPriority()));
 
-    Stream<Long> prioStream = uniqPrios.stream();
+    Stream<Short> prioStream = uniqPrios.stream();
 
     if (uniqPrios.size() > 100) {
       // Until #2094 is addressed limit what is sent to the coordinator to avoid causing it run out
```

Comment:
This comment could probably be removed.  Was not sure how #2094 would be addressed.  With short, thinking we may want to keep 100 limit.

Reply:
I updated the comment in 2d548ab550a2fe3df661d37cb7a120b015014294 to remove the reference to the issue number. I did leave in place the 100 limit.